x-pack/plugins/uptime/public/components/overview/alerts/settings_message_expression_popover.tsx

```diff
@@ -27,7 +27,7 @@ export const SettingsMessageExpressionPopover: React.FC<SettingsMessageExpressio
   id,
 }) => {
   const kibana = useKibana();
-  const path = kibana.services?.application?.getUrlForApp('uptime', { path: 'settings' });
+  const settingsPath = kibana.services?.http?.basePath.prepend('uptime/settings');
```

Comment:
Because alerts are registered in the `setup` function, this component no longer has access to getUrlForApp.